README.md

```diff
@@ -11,21 +11,20 @@ This module configures an [IAM secrets engine](https://cloud.ibm.com/docs/secret
 
 The module supports the following operations:
 
-- Optionally assigning a given user a secrets-manager "Manager" role (disabled by default - not recommended for production - users should be assigned roles using access groups instead of direct role access)."
-- Creates a new ServiceID
-- Assigns the new ServiceID "Editor" role access for "iam-groups"
-- Assigns the new ServiceID "Operator" role access for "iam-identity". If user set the input variable `add_service_id_creator_role` to `true`, this module will also add "Service ID creator" role access for "iam-identity". It is recommended to set it true if the service id creation is disabled in the iam settings.
-- Creates a new apikey for the ServiceID
-- Optionally creates a new secret group (an existing group can be pass in otherwise)
-- Creates a new arbitrary secret in the secret group with the generated ServiceID apikey value (NOTE: This ServiceID apikey secret is stored as an arbitrary secret, meaning the IAM engine will not dynamically create the key or manage rotation of it)
-- Configures the secret manager instance with the IAM engine using the ServiceID apikey
+- Creates a new Service ID
+- Assigns the new Service ID "Editor" role access for "iam-groups"
+- Assigns the new Service ID "Operator" role access for "iam-identity". If user set the input variable add_service_id_creator_role to true, this module will also add "Service ID creator" role access for "iam-identity". It is recommended to set it true if the service ID creation is disabled in the IAM settings.
```

Comment:
I don't understand what iam-groups and iam-identity are. Can we add the descriptor of what they are (...for the `iam-groups` xxx)? Note the definite article `the`. Also wrap them in markdown backticks instead of double quotation marks.